cmd/storage-rpc-client.go

```diff
@@ -93,15 +93,14 @@ func toStorageErr(err error) error {
 }
 
 // Initialize new rpc client.
-func newRPCClient(ep storageEndPoint) (StorageAPI, error) {
-	// Input validation.
-	if ep.host == "" || ep.port == 0 || ep.path == "" {
+func newPosixRPC(ep *url.URL) (StorageAPI, error) {
```

Comment:
calling it `newRPCClient` is more accurate because the function returns a structure related to rpcClient

also hypothetically if the remote storage can be a DB, calling this newRPCClient makes more sense